@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -264,7 +264,7 @@ export interface BaseTechniqueParams {
     minZoomLevel?: DynamicProperty<number>;
 
     /**
-     * Maximum zoom level. If the current zoom level is larger, the technique will not be used.
+     * Maximum zoom level. If the current zoom level is equal to or greater, the technique will not be used.
```

Comment:
You may want to check with our Design team before going forward with this change of semantics.

Reply:
Also, shouldn't you update the semantics of all the zoom/data level constraints? e.g. `DataSource.maxDisplayLevel`, `DataSource.maxDataLevel`, `MapView.maxZoomLevel`, `MapControls.maxZoomLevel`?

Reply:
As agreed for the time being we leave the semantic of those properties as-is, both edges inclusive.